yogstation/code/modules/clothing/masks/hailer.dm

```diff
@@ -8,7 +8,7 @@
 	var/obj/item/radio/radio //For engineering alerts.
 	var/radio_key = /obj/item/encryptionkey/headset_sec
 	var/radio_channel = "Security"
-	var/dispatch_cooldown = 20
+	var/dispatch_cooldown = 250
```

Comment:
```suggestion
	var/dispatch_cooldown = 50
```
Just change it to 5 secs maybe, so it's a bit harder to spam but people can still call multiple crimes in the event of an escalating situation?

Reply:
no

Reply:
Why not? You've had the (IMO) legitimate criticism brought up that raising the cooldown by such a great deal would make it impossible to effectively report escalating situations. If someone's spamming Sec comms with false crimes, why not AHelp it? I'd imagine that spamming Sec comms, to the point that legitimate reports get buried, violates _some_ rule.